packages/devextreme/js/ui/map.d.ts

```diff
@@ -205,6 +205,24 @@ export interface dxMapOptions extends WidgetOptions<dxMap> {
      * @public
      */
     focusStateEnabled?: boolean;
+    /**
+     * @docid
+     * @default { mapId: '', useAdvancedMarkers: true }
+     * @public
+     */
+    googleMapConfig?: {
```

Comment:
We don't want to make this option name too specific because it can be used in the future for other provider's config. So maybe it's better to name it like `providerConfig`?

Reply:
renamed